### ISSUE TITLE
Lack of contrast for accessibility

### ISSUE THREAD

**Issue by eroepken** [eroepken/ellie-strejlau#2]
Lack of contrast between the turquoise color and the gray background can cause people with vision issues to have trouble reading the website. Start by making the turquoise darker and the gray background lighter.

Homepage report: http://cynthiasays.com/report.aspx?id=3035

**Comment by eroepken** [comment]
Re-ran report today for AA compliance.
